Doc/library/sqlite3.rst

```diff
@@ -1444,6 +1421,33 @@ Python types via :ref:`converters <sqlite3-converters>`.
 How-to guides
 -------------
 
+.. _sqlite3-placeholders:
+
+SQL operations usually need to use values from Python variables. However,
```

Comment:
This is missing a section heading, e.g.

```suggestion
.. _sqlite3-placeholders:

Using placeholders in SQLite queries
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

SQL operations usually need to use values from Python variables. However,
```

If you prefer a different one, feel free to use it instead.

Reply:
🤦🏻‍♂️

Reply:
```suggestion
.. _sqlite3-placeholders:

Using placeholders to bind values in SQL queries
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

SQL operations usually need to use values from Python variables. However,
```

Reply:
... or _"Binding values to SQL statements using placeholders"_

Reply:
I find the former a lot more clear